std/experimental/allocator/common.d

```diff
@@ -743,3 +743,135 @@ version(unittest)
         }
     }
 }
+
+/+
+Can the representation be determined at compile time to consist of nothing but
+zero bits? Padding between a struct's fields is not considered.
++/
+private template isAllZeroBits(T, T value)
+{
```

Comment:
Is there a more elegant way than this to check if T.init is entirely zero bits? `typeid(T).initializer() is null` isn't available.